crates/workspace/src/pane.rs

```diff
@@ -105,12 +105,32 @@ pub struct CloseActiveItem {
 #[serde(rename_all = "camelCase")]
 pub struct CloseInactiveItems {
     pub save_intent: Option<SaveIntent>,
+    pub close_pinned: Option<bool>,
 }
 
 #[derive(Clone, PartialEq, Debug, Deserialize, Default)]
 #[serde(rename_all = "camelCase")]
 pub struct CloseAllItems {
     pub save_intent: Option<SaveIntent>,
+    pub close_pinned: Option<bool>,
```

Comment:
```suggestion
    pub close_pinned: bool,
```

Why cannot it be just this? What does `None` mean?

Same for everything below.

Reply:
I opted for an Option to keep support for old keymaps, however if that is not reason enough I can switch it to bool!

(Setting it to a bool would require that every CloseAction would have to be followed by brackets and an explicit boolean value, whereas leaving it empty now would mean close_pinned would be "false" due to the unwrap_or statements).

Reply:
> Setting it to a bool would require that every CloseAction would have to be followed by brackets and an explicit boolean value

Can you elaborate? 
I think it's not true at all, as when we deserialize, we fill in the defaults for the missing declarations.
Example:
we have https://github.com/zed-industries/zed/blob/b0b29d91f9a21870dc48932ae54ad5f812298bd4/assets/keymaps/default-macos.json#L117

and when I declare
```
{
    "bindings": {
      "shift-d": "editor::SelectToBeginningOfLine"
    }
  }
```

it works for me juts fine.

Reply:
Alright, then I was mistaken, will alter it to bool and update the json files. Thanks!